easy_rec/python/layers/sequence_feature_layer.py

```diff
@@ -65,7 +67,10 @@ def negative_sampler_target_attention(self,
 
     concat_features = tf.tile(
         concat_features[:, tf.newaxis, :], multiples=[1, neg_num_add_1, 1])
-    seq_len = tf.tile(seq_len, multiples=[neg_num_add_1])
+
+    # seq_len = tf.tile(seq_len, multiples=[neg_num_add_1])
+    seq_len = tf.tile(seq_len[:, tf.newaxis], multiples=[1, neg_num_add_1])
```

Comment:
when use fg_op, does negative_sampler_target_attention need tf.tile?

Reply:
When using fg_op, there is no need to call negative_sampler_target_attention, just call target_attention.